pkg/engine/mutation.go

```diff
@@ -22,7 +21,11 @@ import (
 )
 
 // Mutate performs mutation. Overlay first and then mutation patches
-func Mutate(ctx context.Context, rclient registryclient.Client, policyContext *PolicyContext) (resp *engineapi.EngineResponse) {
+func Mutate(
```

Comment:
Why are some function headers reformatted and others aren't? What's the rule applied here?

Reply:
No hard rule, when there are too many args, I usually split for better clarity.